proposals/specification/agent-management/ard-response.bs

```diff
@@ -0,0 +1,62 @@
+
+### Discovery Response ### {#sec-ard-response}
+
+Upon receiving a request for an [=Agent Registration=], the [=Agent Manager=] [MUST] first check the authentication provided by the [=Agent=].
+
+
+#### Successful Response #### {#sec-ard-response-success}
+
+If an [:Authorization:] header is present, with the scheme set to [:DPoP:], and
+the included [=Identity Token=] is valid, and it authenticates an [=Agent=] known to
+the [=Agent Manager=], the latter [MUST] respond with either of the following; and it 
```

Comment:
Why do we need those 3 different options? I think `204` is the safest given that it also can act as Identity Document of the Authorization Agent / Agent Manager

Reply:
The `200` allows for optimization, in which the Agent Manager can already respond with the Agent-Specific Resource in the body.

I added the `303` because to me that is the most correct response when used with `pim:preferencesFile` or `rdfs:seeAlso`. In fact, I would drop the other two status codes and prefer the `pim:preferencesFile` predicate over `interop:hasAgentManager`

Reply:
Agreed in meeting to keep the options.